packages/program-boilerplate/src/transaction.ts

```diff
@@ -282,20 +282,24 @@ export default class Transaction {
     rewardKey,
     referralId,
     user,
+    status,
+    rewardProperties,
   }: {
     emailKey: string;
```

Comment:
At this point it might be worth exporting this as a named type instead of this huge anonymous type haha. Other than that looks good.